mkdocs.yml

```diff
@@ -162,7 +162,7 @@ plugins:
               "hc/en-us/articles/207312879.md": "coverage-reporter/index.md"
               "hc/en-us/articles/207312879-Generate-Coverage.md": "coverage-reporter/index.md"
               "hc/en-us/articles/207993605.md": "getting-started/codacy-quickstart.md"
-              "hc/en-us/articles/207993605-I-added-a-repository-Now-what-.md": "getting-started/codacy-quickstart.md"
+              "hc/en-us/articles/207993605-I-added-a-repository-Now-what-.md": "getting-started/tweak-your-settings.md"
```

Comment:
Interesting - I remembered that we used to have a page called "I added a repository. Now what?"

This new page that we're building should address exactly that. :slightly_smiling_face:

Reply:
I like it! It's very descriptive. I added to the PR description as a first candidate for a final title.